optimum/exporters/neuron/config.py

```diff
@@ -53,3 +56,39 @@ class TextAndVisionNeuronConfig(NeuronConfig):
     """
 
     DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator, DummyVisionInputGenerator, DummyBboxInputGenerator)
+
+
+class DecoderNeuronConfig(ExportConfig):
+    """
+    Handles decoder architectures.
+    """
+
+    NEURONX_ARGS = {"batch_size": 1, "n_positions": 128, "tp_degree": 2, "amp": "f32"}
+    NEURONX_MODULE = None
+    NEURONX_CLASS = None
```

Comment:
Maybe we can refactor both like:

```python
NEURONX_CLASS`: str = "gpt2.model.GPT2ForSampling"
```

I do not think that adding the `transformers_neuronx` here is neeeded since it's implied and we can add it once when we load the actual class.

Reply:
I agree that transformers_neuronx could be omitted and inserted just when the class is imported.

I cannot merge the two constants, but I will need to split them back at import, because using `importlib` I cannot import a single symbol without loading the module that contains it entirely. In other words, I cannot do that:
```
module = importlib.import_module(`transformers_neuronx.gpt2.model.GPT2ForSampling)
```
It will raise the following exception:
```
ModuleNotFoundError: No module named 'transformers_neuronx.gpt2.model.GPT2ForSampling'; 'transformers_neuronx.gpt2.model' is not a package
```
This is why I need first to load the module (I load it with importlib), then access the class as an attribute.

Reply:
Yes, I think it's okay to split them:

```python
module_name, attribute_name = self.NEURONX_CLASS.rsplit(".", maxsplit=1)
```

Reply:
Done